scripts/is-release-needed.sh

```
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# This script checks if the current version of the package is already published on npm
+PACKAGE_VERSION="$(node -p -e "require('./package.json').version")"
+PACKAGE_NAME="$(node -p -e "require('./package.json').name")"
+
+FOUND_VERSION=$(npm view $PACKAGE_NAME versions | grep $PACKAGE_VERSION)
```

Shellcheck

[info] Double quote to prevent globbing and word splitting.

(SC2086)